{"name": "test_change_not_auth_user_fail", "status": "passed", "description": "Изменение данных неавторизованного пользователя", "start": 1720286487050, "stop": 1720286488345, "uuid": "3fee794d-0ae7-408d-bddd-8f3b4e9bf84e", "historyId": "573686f6ec4567847eb8e31d3d321bed", "testCaseId": "573686f6ec4567847eb8e31d3d321bed", "fullName": "tests.test_change_user_data.TestChangeUserData#test_change_not_auth_user_fail", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_change_user_data"}, {"name": "subSuite", "value": "TestChangeUserData"}, {"name": "host", "value": "DESKTOP-LU3QF62"}, {"name": "thread", "value": "19692-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_change_user_data"}]}